dodola/cli.py

```diff
@@ -191,3 +191,18 @@ def regrid(x, out, method, domain_file, weightspath):
         domain_file=domain_file,
         weights_path=weightspath,
     )
+
+
+@dodola_cli.command(help="Correct wet day frequency in a dataset")
+@click.argument("x", required=True)
+@click.option("--out", "-o", required=True)
+@click.option(
+    "--process",
+    "-p",
+    required=True,
+    type=click.Choice(["pre-process", "post-process"], case_sensitive=False),
+    help="Whether to pre or post process wet day frequency",
+)
+def correct_wetday_frequency(x, out, process):
+    """Correct wet day frequency in a dataset"""
+    services.correct_wet_day_frequency(str(x), out=str(out), process=str(process))
```

Comment:
Potentially bigger issue with implications across files:

Do we need get a user input variable name, and extract/correct a DataArray for that variable, returning a Dataset with the corrected variable?

As it is right now, couldn't this correction apply to everything in the input Dataset? (and that's potentially a disaster...?)

Reply:
Right now it is up to the user to only apply this to precip. Do we want to raise an error if the variable in a dataset is _not_ precip?

Reply:
Ah, @dgergel , first I was thinking that there could be an unexpected side effect if this filtered a coordinate variable (lat, lon, height) ...but I couldn't reproduce it with a reasonable bug in a quick test, so I'll drop it. That's bogus.

The other case where this might be a problem is if someone kept temp and precip in the same zarr store. The user would be able to say "yo, only do this for the `precip` variable" -- so that it wouldn't filter the temp data available in the same store.

This is a bit of a reach so I wouldn't change it unless you see the need.